.gitignore

```diff
@@ -1,3 +1,52 @@
+# Byte-compiled / optimized / DLL files
 __pycache__/
-*.idea
-*.DS_store
+*.py[cod]
+*$py.class
+
+# Mac/OSX
+.DS_Store
+
+# Windows
+Thumbs.db
+
+# PyCharm
+.idea/
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+pip-wheel-metadata/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# Unit test / coverage reports
+.coverage
+.coverage.*
+.cache
+.pytest_cache/
+
+# Sphinx documentation
+docs/_build/
+
+# Environments
```

Comment:
These can be removed. They all depend on the user choosing to name the folder as such, and place it in the root of the repo.